tests/metrics/network/lib/network-test-common.bash

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# This file contains functions that are shared among the networking
+# tests that are using nuttcp and iperf
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# This function will setup cor as a runtime, clean up the
+# environment and create a temporary file where networking
+# results will be stored
+function setup {
+	runtime_docker
+	kill_processes_before_start
+	result=$(mktemp)
+}
+
+# This function can be used in iperf and nuttcp networking tests
+function start_server {
+	local server_name="$1"
+	local image="$2"
+	local server_command="$3"
+	$DOCKER_EXE run -d --name=${server_name} ${image} sh -c "${server_command}" > /dev/null
+	server_address=$($DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" ${server_name})
```

Comment:
This is looking better with the `local` variables, but the function is still setting a global variable. My previous comment was suggesting that this function could end with:

```
echo "$server_address"
```

... meaning you could then call,

```
server_address=$(start_server ...)
```

The script is now partially using locals and partially using globals. I think for consistency you should try to ensure that none of the functions (maybe with the exception of `setup` and `clean_environment`) use globals. If that just isn't possible, it would be a good idea to document on each function the globals its using (and why).